helix-core/src/test/java/org/apache/helix/integration/task/TestBatchAddJobs.java

```diff
@@ -86,6 +86,9 @@ private String getPrefix(String job) {
 
   @AfterClass
   public void afterClass() {
+    String testClassName = this.getShortClassName();
+    System.out.println("AfterClass: " + testClassName + " of TestBachAddJobs called.");
```

Comment:
Humm, testClassName should be "TestBachAddJobs", right? So it will be "AfterClass: TestBachAddJobs of TestBachAddJobs called."?

Moreover, why we add this logic to every child test class instead of print in the parent class?

Reply:
Changed to  System.out.println("AfterClass: " + testClassName + " called.");

Now changed to print in parent class, as suggested.